app/js/lib/server/reminders.js

```diff
@@ -41,6 +41,17 @@ export default class Reminders {
   }
 
   /**
+   * Create a new reminder.
+   *
+   * @param {Object} body
+   * @return {Promise}
+   */
+  update(body) {
+    const id = body.id;
```

Comment:
A couple of comments here, should we check for the existence of `body.id` here and reject.

Also, this method is quite similar to the `set(body)` one, as we send the whole body, should be the api work in that way?